common/src/commonMain/kotlin/com/studo/campusqr/common/payloads/CampusQRPayloads.kt

```diff
@@ -0,0 +1,15 @@
+package com.studo.campusqr.common.payloads
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+class UserData(
+  var appName: String,
+  var clientUser: ClientUser? = null, // Null when unauthenticated
+  var externalAuthProvider: Boolean = false,
+  var liveCheckInsViewEnabled: Boolean,
+) : ClientPayload
+
+interface ClientPayload
```

Comment:
move to top of file, maybe add description that this is used to mark payloads that are sent to the client and also add that they must be `@Serializable`